### ISSUE TITLE
Cannot speech whitelist textarea with NVDA screen reader

### ISSUE THREAD

**Issue by riku22** [uBlockOrigin/uBlock-issues#423]
### Prerequisites

- [ ] I verified that this is not a filter issue
    - Filter issues MUST be reported at [filter issue tracker](https://github.com/uBlockOrigin/uAssets/issues)
    
- [ ] This is not a support issue or a question
    - Support issues and questions are handled at [/r/uBlockOrigin](https://old.reddit.com/r/uBlockOrigin/)
    
- [ ] I performed a cursory search of the issue tracker to avoid opening a duplicate issue
    - Your issue may already be reported.
- I tried to reproduce the issue when...
    - [ ] uBlock Origin is the only extension
    - [ ] uBlock Origin with default lists/settings
    - [ ] using a new, unmodified browser profile
- [ ] I am running the latest version of uBlock Origin
- [ ] I checked the [documentation](https://github.com/gorhill/uBlock/wiki) to understand that the issue I report is not a normal behavior

### Description

I use [NVDA](https://www.nvaccess.org/") software which reads character information on the screen of my computer.
However, NVDA can not make the text area of the whitelist read correctly.

### Steps to Reproduce

1. Install NVDA.
1. Open the UBlock setting screen.
1. Open a whitelist.
1. Focus on the text field where the whitelist is written.
1. Press the up arrow, down arrow, left arrow, or right arrow, to speak.

### Expected behavior:

- When I press the up arrow or down arrow, it reads as "about-scheme".
- When I press the left arrow or right arrow, it reads as "a", "b", "o".

### Actual behavior:

It reads it as "blank".

### Remarks
In older versions, I read it correctly.

### Your environment

* uBlock Origin version: 1.18.4
* Browser Name and version: Google Chrome 72.0.3626.109
* Operating System and version: Windows10 1809 64bit

**Comment by uBlock-user** [comment]
Something to do with the translation perhaps..

**Comment by riku22** [comment]
I switched Chrome's language to English and tried it, but it did not change.

**Comment by uBlock-user** [comment]
Does it occur on Firefox ?

**Comment by gorhill** [comment]
Probably related to using CodeMirror. CodeMirror's issue tracker this will need to be investigated to find out if such issue has been reported.

**Comment by riku22** [comment]
> Does it occur on Firefox ?

 No.

**Comment by DimitrisKalaboukas** [comment]
@gorhill Also, refer to https://github.com/nvaccess/nvda/issues/8691.

> I close the issue as unrelated to NVDA, waiting for new version of CodeMirror library with improved accessibility, as reported by @Neurrone. Until that moment, obviously the problem still occurs, after release I consider if open this issue again.

**Comment by krystian3w** [comment]
> new version of CodeMirror library with improved accessibility

It will this fix be incorporated into raw CodeMirror 5?
